lib/elixir_conf_africa/ticket_types/ticket_type.ex

```diff
@@ -0,0 +1,23 @@
+defmodule ElixirConfAfrica.TicketTypes.TicketType do
+  @moduledoc false
+
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "ticket_types" do
+    field :name, :string
+    field :description, :string
+    field :price, :decimal
+    belongs_to :event, ElixirConfAfrica.Events.Event
+
```

Comment:
Nice work @lgmfred  , I would suggest here we add the column number that accounts for how many such tickets will be available , for example , we may have only 100 VIP tickets depending on the venue